gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js

```diff
@@ -75,6 +75,19 @@ https://github.com/gpii/universal/LICENSE.txt
         ]
     };
 
+    gpii.ontologyHandler.tests.floydWarshallAlgorithmWithReconstruction = function () {
+        var edges = [["A","B"], ["B","A"], ["C","B"], ["B", "C"], ["B","D"], ["D", "B"]];
```

Comment:
We should fix the algorithm so that it implicitly assumes an undirected graph (as well as its current assumption of weight 1)